src/main/java/org/ibp/api/rest/dataset/DatasetResource.java

```diff
@@ -565,6 +566,24 @@ public ResponseEntity<List<MeasurementVariable>> getStudyVariablesByVariableType
 		return new ResponseEntity<>(variables, HttpStatus.OK);
 	}
 
+	@ApiOperation(value = "Get Observation Audit", notes = "Update Observation")
+	@PreAuthorize("hasAnyAuthority('ADMIN','STUDIES', 'MANAGE_STUDIES')")
+	@RequestMapping(value = "/{crop}/programs/{programUUID}/studies/{studyId}/datasets/{datasetId}/"
+		+ "observationUnits/{observationUnitId}/variable/{variableId}/phenotype-audit", method = RequestMethod.GET)
+	public ResponseEntity<List<PhenotypeAuditDTO>> getPhenotypeAudit(
```

Comment:
Move this to a new resource class "ObservationAuditResource".
Change the request mapping to:
/crops/{cropName}/observationUnits/{observationUnitId}/variable/{variableId}/changes